pandas/tests/frame/methods/test_value_counts.py

```diff
@@ -100,3 +100,46 @@ def test_data_frame_value_counts_empty_normalize():
     expected = pd.Series([], dtype=np.float64)
 
     tm.assert_series_equal(result, expected)
+
+
+def test_data_frame_value_counts_dropna_true():
+    df = pd.DataFrame(
+        {
+            "first_name": ["John", "Anne", "John", "Beth"],
+            "middle_name": ["Smith", pd.NA, pd.NA, "Louise"],
```

Comment:
Please parametrize both tests over all NaN values (we have a fixture for this)

Reply:
Like this? aa94b3b

Reply:
Yes thx